connector_salesforce/README.rst

```diff
@@ -0,0 +1,247 @@
+|                                  |           | with the checkbox IsCompany    |
+|                                  |           | unchecked linked to            |
+|                                  |           | the parent partner             |
++----------------------------------+-----------+--------------------------------+
+| State + Country + title          |     →     | State + Country + title        |
+|                                  |           |                                |
++----------------------------------+-----------+--------------------------------+
+| Product2                         |   ← or →  | Product                        |
+|                                  |           | Import of product was desing   |
+|                                  |           | for initial import and should  |
+|                                  |           | not be used latter even if it  |
+|                                  |           | can work                       |
++----------------------------------+-----------+--------------------------------+
+| Pricebook entry                  |     →     | Pricelist item                 |
+|                                  |           |                                |
++----------------------------------+-----------+--------------------------------+
+| Won Opportunity                  |     →     | Sale Order                     |
++----------------------------------+-----------+--------------------------------+
+
+Test UseCase
+------------
+
+In Salesforce
+*************
+
+Creation of an Account
+######################
+
+ * Scroll down to see the section ‘Contact’ and click on ‘New contact’
+   - create contact ‘Mrs Marie Durand’
+   - create contact ‘Helmut’
+
+Creation of an opportunity
+##########################
+
+ * Creation of the ‘opportunity A’
+   - Scroll down to see the section ‘Quote’ and click on ‘New quote’
+
+ * Creation of the Quote A1
+   - with contact ‘Helmut’ (which does not belong to the Company A)
+ * save
+
+ * Scroll down to see the section ‘Quote Line items’ and click on ‘Add Line item’ in order to add products in the quote.
+ * Tick the product and click on Select button
+
+ * Change the price to 99, set the Qty = 5 and add a discount of 15%
+
+ * Save
+
+Set the Quote/Opportunity as Won
+################################
+
+ * Go back to the Quote, click on ‘Start Synch’ (and confirm)
+
+ * Reopen the opportunity, click on Edit and change the stage to ‘Won’
+
+ * Save
+
+In Odoo
+-------
+
+Now the Opportunity can be sync with Odoo.
+
+ * Import won opportunity from Odoo. The import of  Opportunity using will automatically trigger all other synchronization. Indeed, for instance, if you try to import an opportunity but the partner (= SF account) does not exist in Odoo then the connector will launch a direct (no job creation) import of this specific account. The same goes for the other missing assets like product, price, etc.
```

Comment:
* Import won opportunity from Odoo. Importing Opportunity in Odoo will automatically trigger all other synchronizations. For instance, if you try to import an opportunity but the partner (= SF account) does not exist yet in Odoo, the connector will automatically launch a direct import of this specific account, without job creation. Same goes for the other missing items such as product, price, etc.